src/app/settings/settings.component.html

```diff
@@ -36,110 +36,13 @@ <h5>Pamja e Zgjedhur</h5>
                                 }
                             </div>
                             <div
-                                class="theme-selector"
+                                class="theme-selector theme-selector-settings"
                                 [ngClass]="{
-                                    'theme-selector-dark-mode':
+                                    'theme-selector-dark-mode theme-selector-settings-dark-mode':
```

Comment:
`theme-selector-settings` and `theme-selector-settings-dark-mode` seem to be declared in the `app-dark-mode-buttons` component instead, why is that?